### PR TITLE
PT-204 | Support line breaks on event description

### PR DESCRIPTION
**Pull request by jorilindell**

### CODE REVIEW

File: src/common/components/textWithLineBreaks/__tests__/TextWithLineBreaks.test.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { shallow } from 'enzyme';
```
Comment (Sartonon) [inline src/common/components/textWithLineBreaks/__tests__/TextWithLineBreaks.test.tsx:1]: why enzyme?

File: src/common/components/textWithLineBreaks/__tests__/TextWithLineBreaks.test.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+import * as React from 'react';
```
Comment (Sartonon) [inline src/common/components/textWithLineBreaks/__tests__/TextWithLineBreaks.test.tsx:2]: why import * as React from 'react'?

Comment (jorilindell) [reply src/common/components/textWithLineBreaks/__tests__/TextWithLineBreaks.test.tsx]: eslint is showing following error "React must be in scope when using JSX" if React is not imported

File: src/common/components/textWithLineBreaks/__tests__/TextWithLineBreaks.test.tsx (hunk 1 above)
Comment (jorilindell) [reply src/common/components/textWithLineBreaks/__tests__/TextWithLineBreaks.test.tsx]: Replaced with @testing-library/react

**Pull request merged**